examples/bare_reduce.c

```diff
@@ -17,9 +19,14 @@ int main(int argc, char **argv)
     MIMPI_Init(false);
     int const world_rank = MIMPI_World_rank();
 
+    const char *delay = getenv("DELAY");
+    if (delay)
+        setenv(WRITE_VAR, delay, true);
+
```

Comment:
Calls to `setenv`, `getenv`, `unsetenv` should be error-checked.

Reply:
Finally I used simple asserts to check for system calls success to not copy whole ASSERT_SYS_OK functionality.